src/styles/colors.scss

```diff
@@ -1,2 +1,14 @@
 $access-green: #1f9614;
 $off-white: #FFFFFE;
+$onelogin-blue: #00A9E0;
+$onelogin-black: #1C1F2A;
+$onelogin-gray: rgb(243, 243, 244);
+
+.onelogin-input {
```

Comment:
I think this needs to go to forms.scss. You just need colors definitions here

Reply:
done